.packit.yaml

```diff
@@ -1,22 +1,23 @@
 specfile_path: ansible-bender.spec
 actions:
   create-archive:
-    - "python3 setup.py sdist --dist-dir ."
-    - "sh -c 'echo ansible-bender-$(python3 setup.py --version).tar.gz'"
+    - "python3 -m build"
+    - "sh -c 'echo dist/ansible-bender-$(python3 -m setuptools_scm).tar.gz'"
   get-current-version:
-    - "python3 setup.py --version"
+    - "python3 -m setuptools_scm"
```

Comment:
This may need piping through `awk` because it may output extra text.

Reply:
yes, that's possible and it already happened: the build fails then and needs investigation